auxx/src/Command/Update.hs

```diff
@@ -95,17 +93,17 @@ propose sendActions ProposeUpdateParams{..} = do
     let BlockVersionData {..} = genesisBlockVersionData
```

Comment:
Do we need this now? What for? If we only need to override some values, do we need any of the original genesis block values?

Reply:
No, that line was unused, I removed it. Good catch.